script.js

```diff
@@ -2,6 +2,224 @@ const unionButton = document.querySelector('.union-button');
+    <ul class="page flex-row">
+      <li class="title_1 ignore-mbl">${project.company}</li>
+      <li class="title_1 ignore-desk">${project.company1}</li>
+      <li><img src="${project.icon}" alt="Dot icon" ></li>
+      <li class="ignore-mbl">${project.specialization}</li>
+      <li class="ignore-desk">${project.specialization1}</li>
+      <li><img src="${project.icon}" alt="Dot icon" ></li>
+      <li>${project.year}</li>
+    </ul>
+    <p class="Text">
+    ${project.description}
+    </p>
+    <ul class="other-tags flex-row primarytext33">
+        <li class="box"><span>${project.technologies.tech1}</span></li>
+        <li class="box_desk "><span>${project.technologies.techE}</span></li>
+        <li class="box"><span>${project.technologies.tech2}</span></li>
+        <li class="box"><span>${project.technologies.tech3}</span></li>
+    </ul>
+    <button type="button" id="${project.id}" class="flex-row see">See Project</button>
+    </div>
+    
+    </div>`;
+  if (count === 2) {
+    count -= 1;
+  } else {
+    count += 1;
+  }
+  cardSection.append(newDiv);
+});
+
+const seeproject = document.querySelectorAll('.see');
+const overlay = document.getElementById('overlay1');
+const popup = document.querySelector('.popup');
+const over = document.querySelector('.over');
+seeproject.forEach((p) => p.addEventListener('click', (p) => {
+  const { id } = p.target;
+  const pop = projects.find((p) => p.id === id);
+  popup.innerHTML = `
+  <div class="titles">
+    <div class="flex-row" style="justify-content:space-between;">
+      <h2 class="tonic-1 ignore-mbl" style="padding-left:6%">${pop.title}</h2>
+      <h2 class="tonic-1 tonic-12  ignore-desk">${pop.title1}</h2>
+      <button class="close close12" style="border-color:white;">&times;</button>
+    </div>
+    <div class="Card_text flex-column">
+    <ul class="page page12 flex-row" style="padding-left:7%">
+      <li class="title_1 ignore-mbl">${pop.company}</li>
+      <li class="title_1  ignore-desk">${pop.company1}</li>
+      <li><img src="${pop.icon}" alt="Dot icon" ></li>
+      <li class="ignore-mbl">${pop.specialization}</li>
+      <li class="ignore-desk">${pop.specialization1}</li>
+      <li><img src="${pop.icon}" alt="Dot icon" ></li>
+      <li>${pop.year}</li>
+    </ul>
+    </div>
+    <img src="${pop.imageMobile}" alt="live" class="work" style="padding-left:6%; margin-top:0%">
+    <img src="${pop.imageDesktop}" alt="live" class="pic"  style="width:1256px; height:440px; margin: 20px;">
+    <div class="explain">
+      <p class="tonic-text2">
+        ${pop.description1}
+        <br class="ignore-desk">
+        <br class="ignore-desk">
+        <span class="ignore-desk">${pop.description1}</span>
+      </p>
+      <div class="smart">
+          <ul class="other-tag flex-row" style="padding-left:6%">
+            <li class="box">${pop.technologies.tech1}</li>
+            <li class="box_desk ignore-mbl">${pop.technologies.techE}</li>
```

Comment:
In order to make your design match the template, I recommend paying close attention to the width of the image as it is currently oversized when compared to that of the template.

<img width="854" alt="image" src="https://github.com/zuhebahmed88091/Portfolio_site/assets/84930355/594d86bc-3a07-42f4-8c97-fb69c2ff285b">

figma design

<img width="344" alt="figma" src="https://github.com/zuhebahmed88091/Portfolio_site/assets/84930355/eeacf4a9-55e3-4e26-89fe-35cb7096fa7a">